.gitignore

```diff
@@ -6,3 +6,4 @@ rdoc/*
 Gemfile.lock
 api_keys.yml
 test/geocoder_test.sqlite
+.idea/*
```

Comment:
`.idea` is created by RubyMine which is a very popular IDE

Reply:
Yes, however, everything in this `.gitignore` file should be specific to the Geocoder gem. To ignore files related to local development, please use your `~/.gitignore_global` file.